src/main/kotlin/pt/up/fe/ni/website/backend/model/Post.kt

```diff
@@ -0,0 +1,16 @@
+package pt.up.fe.ni.website.backend.model
+
+import java.util.Date
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+
+@Entity
+class Post(
+    var title: String,
+    var body: String,
+    var href: String,
+    var templatePath: String,
```

Comment:
```suggestion
    var thumbnailPath: String,
```
My bad, I think it was my typo in the other review